package.json

```diff
@@ -5,15 +5,21 @@
   "main": "index.js",
   "scripts": {
     "start": "node index",
-    "dev": "nodemon index"
+    "dev": "nodemon index",
+    "test": "mocha"
   },
   "author": "AlexAlt",
   "license": "ISC",
   "devDependencies": {
     "nodemon": "^3.1.7"
   },
+  "type": "module",
   "dependencies": {
+    "chai": "^5.1.1",
```

Comment:
I should put these as dev dependencies but this PR is already getting super long so I might do a cleanup branch later